modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -29,7 +29,9 @@ def show
           end
 
           output = generate_show_output(bgs_claim: bgs_claim, lighthouse_claim: lighthouse_claim)
-          blueprint_options = { base_url: request.base_url, veteran_id: params[:veteranId] }
+          blueprint_options = { base_url: request.base_url, veteran_id: params[:veteranId],
+                                ptpct_id: target_veteran.participant_id }
```

Comment:
When you move that logic out of the blueprint, you won't need this change anymore.